src/protocols/secure_channel/CASESession.cpp

```diff
@@ -974,140 +1074,143 @@ CHIP_ERROR CASESession::TryResumeSession(SessionResumptionStorage::ConstResumpti
 CHIP_ERROR CASESession::HandleSigma1(System::PacketBufferHandle && msg)
 {
     MATTER_TRACE_SCOPE("HandleSigma1", "CASESession");
+    ChipLogProgress(SecureChannel, "Received Sigma1 msg");
+    MATTER_TRACE_COUNTER("Sigma1");
+
     CHIP_ERROR err = CHIP_NO_ERROR;
     System::PacketBufferTLVReader tlvReader;
 
-    uint16_t initiatorSessionId;
-    ByteSpan destinationIdentifier;
-    ByteSpan initiatorRandom;
+    // Struct that will serve as output in ParseSigma1
+    ParsedSigma1 parsedSigma1;
 
-    ChipLogProgress(SecureChannel, "Received Sigma1 msg");
-    MATTER_TRACE_COUNTER("Sigma1");
+    ReturnErrorOnFailure(mCommissioningHash.AddData(ByteSpan{ msg->Start(), msg->DataLength() }));
```

Comment:
Move this above parsedSigma1. Let's keep initializations closest to definition points.

Reply:
Done